python/hail/methods/statgen.py

```diff
@@ -196,8 +197,8 @@ def impute_sex(call, aaf_threshold=0.0, include_par=False, female_threshold=0.2,
     ------
     :class:`.Table`
         Sex imputation statistics per sample.
-
     """
+    assert(aaf_threshold >= 0.0 and aaf_threshold <= 1.0)
```

Comment:
Sorry, should have been clearer what I wanted. Something like this. Not sure what error to raise.
```
if not 0.0 <= aaf_threshold <= 1.0:
    raise FatalError("invalid argument for `aaf_threshold`. Must be in range [0, 1].")
```